### ISSUE TITLE
Extension Doesnt Support OSS Variant

### ISSUE THREAD

**Issue by LC43** [shanalikhan/code-settings-sync#77]
**Visual Studio Code Version :** [1.4.0  ]
**Code Sync Settings Version :** [ 2.0.6 ]
**Operating System :** [ linux ]

Hi, I have a machine A with vscode - Open Sourse ( OSS ) and another B with default vscode.

On machine A

the configs are in: `~/.config/Code - OSS/`

and extensions on: `~/.vscode-oss/`

---

On Machine B:

the configs are in: `~/.config/Code/`

and extensions on: `~/.vscode/`

Anyway, this will change soon, as per [#3884](https://github.com/Microsoft/vscode/issues/3884) but it will, maintain the -variant, part.

Hope you have time to look this up!

thanks!

**Comment by shanalikhan** [comment]
Thanks, I will work on it.

**Comment by shanalikhan** [comment]
There are officialy two variants of Code - Insiders and stable.
I think extension should be limited to those two only.

**Comment by rainbyte** [comment]
I've found this issue while trying to sync to machines with vscode-oss installed.

Will oss version be supported later?

**Comment by shanalikhan** [comment]
May be it will be supported later
But i asked the Code team on https://github.com/Microsoft/vscode/issues/13935
and their response was 
>But those are the only two variants.

Stable and Insiders.

**Comment by rainbyte** [comment]
Thanks for taking time to handle the issue.

For now I've used this as workaround:
```
ln -s ~/.vscode-oss ~/.vscode
ln -s ~/.config/Code\ -\ OSS ~/.config/Code
```

It worked at least, but I suppose is not a good practice

**Comment by shanalikhan** [comment]
https://github.com/shanalikhan/code-settings-sync/pull/479

**Comment by shanalikhan** [comment]
Released with v2.9
let me know if there are issues

**Comment by jlnr** [comment]
I've installed the flatpak version from flathub, and I needed to create this symlink as a workaround:
```bash
ln -s ~/.var/app/com.visualstudio.code/config/Code ~/.config/Code
```
